contents/blog/product-market-fit-game.md

```diff
@@ -0,0 +1,351 @@
+
+### Level 3 - Validate your MVP by recruiting real users
+
+//TODO: Level 3 image here
+
+You should start Level 3 as soon as you have something for users to use. You need to _keep_ talking to users throughout this level, but now you can validate better because you have a product they can see.
+
+The goal here is to see if people actually care, as quickly as possible. No matter how excited you get after interviewing people about their problems, there's no substitute for seeing what people _do_ instead of what they _say_. 
+
+With one of the ideas Tim and I worked on before we pivoted to PostHog, we had 15 companies who "wanted to try it". We built a product, sent the invite link, and only _one_ even clicked the link and started using it. It's a good thing we hadn't spent 6+ months building the "perfect" product before this happened!
+
+The most obvious users to start with here are those that you've already met during levels 1 and 2. You should have been identifying a _problem_ that these users have during those calls. Now you have a _solution_, you should email them saying so.
+
+#### Failure modes
+
+These are all the reasons why users may not use your product:
+
+* **Users have no idea what your solution is**, so they think it could be a waste of time. Even if your product is free, people's time is not. The more senior your likely users, the stronger this effect. Can you work with someone that _reports_ to the person you were targeting to get things setup? If you struggle to get responsiveness, start questioning if something else is causing issues – even a senior stakeholder will put in work to solve a serious problem they have.
+
+- **You're not explaining your product clearly.** The way you explain your product matters. If it's complicated, people won't understand it. Describe it like you would when speaking to a friend. Ask users if the above is why they didn't start using it. Explain that you are seeking direct feedback in your message, or on a call.
+
+- **You lack credibility.** Businesses won't take a chance on you if your website is a mess, or you appear disorganized. At best, it'll be a waste of time for them; at worst, your product may be insecure, or cause them issues. Hopefully, during Level 1, you picked this idea because you've encountered it personally. That might mean you're well-qualified to solve it for others. Clarify your credentials, if relevant and impressive, when you interact with them. People (mostly) won't voice their fears, so they may go silent or, worse, give you misleading feedback that it's something else.
+
+* **Users are sceptical your solution will solve the problem**, so there's no point trying. Potential customers are unlikely to try your product if they disagree with your approach. Possible failure modes here include statements like "software can't solve a people problem" or "AI looks cool, but won't work in real-life applications because it's not accurate enough".
```

Comment:
```suggestion
* **Users are skeptical your solution will solve the problem**, so there's no point trying. Potential customers are unlikely to try your product if they disagree with your approach. Signs here include statements like "software can't solve a people problem" or "AI looks cool, but won't work in real-life applications because it's not accurate enough".
```